libs/database/src/lib/services/content.service.ts

```diff
@@ -1294,4 +1294,115 @@ Message: ${message}`,
       items: result,
     };
   };
+
+  /**
+   *
+   * @param {string} userId
+   * @param {number} maxResults
+   * @param {string} sinceId
+   * @param {string} untilId
+   * @returns
+   */
+  getEngagementFromUser = async (
+    userId: any,
+    maxResults: number = DEFAULT_CONTENT_QUERY_OPTIONS.maxResults,
+    sinceId: string,
+    untilId: string
+  ) => {
+    let filter: FilterQuery<Engagement> = {
+      type: 'like',
+      user: userId,
+      visibility: EntityVisibility.Publish,
+    };
+    const totalDocument = await this._engagementModel
+      .countDocuments(filter)
+      .exec();
+
+    filter = await createCastcleFilter(filter, {
+      sinceId: sinceId,
+      untilId: untilId,
+    });
+
+    const result = await this._engagementModel
```

Comment:
```suggestion
    const engagements = await this._engagementModel
```

ตั้งชื่อให้ชัดเจนหน่อยครับ พวก result, data, items, payload ควรเลี่ยง
guideline: https://dev.to/somedood/a-grammar-based-naming-convention-13jf